bcda/responseutils/writer.go

```diff
@@ -22,4 +24,91 @@ func WriteError(outcome *fhirmodels.OperationOutcome, w http.ResponseWriter, cod
 	}
 }
 
-func WriteCapabilityStatement() {}
+func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *fhirmodels.CapabilityStatement {
+	usecors := true
+	bbServer := os.Getenv("BB_SERVER_LOCATION")
+	statement := &fhirmodels.CapabilityStatement{
+		Status:       "active",
+		Date:         &fhirmodels.FHIRDateTime{Time: reldate, Precision: fhirmodels.Date},
+		Publisher:    "Centers for Medicare & Medicaid Services",
+		Kind:         "capability",
+		Instantiates: []string{bbServer + "/baseDstu3/metadata/"},
+		Software: &fhirmodels.CapabilityStatementSoftwareComponent{
+			Name:        "Beneficiary Claims Data API",
+			Version:     relversion,
+			ReleaseDate: &fhirmodels.FHIRDateTime{Time: reldate, Precision: fhirmodels.Date},
+		},
+		Implementation: &fhirmodels.CapabilityStatementImplementationComponent{
+			Description: "",
+			Url:         baseurl,
+		},
+		FhirVersion:   "3.0.1",
```

Comment:
If we are going to hardcode this `FhirVersion`, maybe we need to create a constant that can be referenced by other apps?  Do we envision this to change frequently?  If so, we might need this to be an env var.

Reply:
Not sure about how often the fhir version will change, would like @rnagle feedback on this

Reply:
This value is not likely to change often. We can shield ourselves from providing the wrong or outdated value by using the corresponding value from the BB 2.0 backend metadata endpoint.

Reply:
yep that's exactly where I took it from

Reply:
I guess what I mean is, if we fetch the backend's capability statement here, we can use its `FhirVersion` and not have to worry about updating this manually.

Reply:
true, but imho I think that's overkill for this especially since relying on bb metadata endpoint for data might not be the most accurate thing to do. Some of the data in their own capability statement is "shaky" like broken url's for example. But I'm not opposed to doing this if that's what we want to do.

Reply:
@tbellj Did you document the issues you found? If not, could you please? Ideally in a JIRA ticket.